docs/targets/http.md

```diff
@@ -0,0 +1,72 @@
+# Event Target for HTTP
+
+This event target receives [CloudEvents][ce] and turns them into HTTP request that consume external services.
+
+## Prerequisites
+
+The HTTP event target sends requests to arbitrary URLs and wraps responses in CloudEvents back to the caller. HTTP endpoints that are unauthenticated, use basic authentication or use custom header values for authentication, can be integrated using this target.
+
+Responses from external HTTP endpoints are converted into [CloudEvents][ce] and sent as a reply to the TriggerMesh Broker/Channel. It is important that the HTTP target filters received events and cares about response event type and event source to avoid loops where those responses might end up being processed by the HTTP Target.
```

Comment:
do we really need to link to `CloudEvents` page every time it;s mentioned?

Reply:
probably not, I'll set it only once at the first mention and remove all other links if that's ok.

Reply:
https://github.com/triggermesh/docs/pull/78/commits/5693292ed04de6fcca7ba68fd64c891171deca06